apko/BUILD.bazel

```diff
@@ -24,18 +24,6 @@ resolved_toolchain(
     visibility = ["//visibility:public"],
 )
 
-bzl_library(
-    name = "repositories",
```

Comment:
Was this change required ? If yes, what is the explanation for it?

Reply:
It was required due to a build breakage, but it looks like I rebased past a fix so it's not needed anymore